gpg-interface.c

```diff
@@ -1019,6 +1021,7 @@ static int sign_buffer_ssh(struct strbuf *buffer, struct strbuf *signature,
 		ssh_signing_key_file = strbuf_detach(&key_file->filename, NULL);
 	} else {
 		/* We assume a file */
+		literal_ssh_key = 0;
```

Comment:
As we initialize `literal_ssh_key` to zero when we declare it I don't think we need this line.

Reply:
Removed that line 👍